Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3698,17 +3705,24 @@ def get_modified_remote_data_command(client: Client, params: dict[str, str],
     """
     ctx, ctx_version = get_integration_context_with_version()
     remote_args = GetModifiedRemoteDataArgs(args)
+
     highest_fetched_id = ctx.get(LAST_FETCH_KEY, 0)
     limit: int = int(params.get('mirror_limit', MAXIMUM_MIRROR_LIMIT))
     fetch_mode = params.get('fetch_mode', '')
     range_ = f'items=0-{limit - 1}'
-    last_update_time = ctx.get(LAST_MIRROR_KEY, 0)
-    if not last_update_time:
-        last_update_time = remote_args.last_update
-    last_update = get_time_parameter(last_update_time, epoch_format=True)
-    assert isinstance(last_update, int)
-    filter_modified = f'id <= {highest_fetched_id} AND status!=closed AND last_persisted_time > {last_update}'
-    filter_closed = f'id <= {highest_fetched_id} AND status=closed AND close_time > {last_update}'
+    last_update_modified = ctx.get(LAST_MIRROR_KEY, 0)
+    if not last_update_modified:
+        # This is the first mirror. We get the last update of the latest incident with a window of 5 minutes
+        last_update = dateparser.parse(remote_args.last_update)
+        if not last_update:
+            last_update = datetime.now()
+        last_update -= timedelta(minutes=5)
+        last_update_modified = int(last_update.timestamp() * 1000)
+    last_update_closed = ctx.get(LAST_MIRROR_CLOSED_KEY, last_update_modified)
```

Comment:
Maybe first check `if LAST_MIRROR_CLOSED_KEY in ctx`,
and only if it's not, then do the whole `last_update` and `last_update_modified` calculation(s)?